CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ can be found at [#317](https://github.com/grafana/agent/issues/317).
 
 - [ENHANCEMENT] Go 1.16 is now used for all builds of the Agent. (@rfratto)
 
+- [ENHANCEMENT] Add support for running as a windows service (@mattdurham)
```

Comment:
I'd say this big enough to be a feature 🤷 It's a pretty arbitrary decision though.